dev-tools/reconfigurator-cli/tests/input/target-release.txt

```diff
@@ -0,0 +1,68 @@
+# Load example system
+load-example --nsleds 3 --ndisks-per-sled 3
```

Comment:
Since you're not testing any of the zone stuff, I believe, you may be able to make this diff much smaller by passing in `--no-zones`.

Reply:
That's true but I don't view the size of this file as a major problem.  I'd rather minimize divergence from an ordinary system.  I do think we should really make `blueprint-diff` skip over stuff that hasn't changed though.